mysql-test/suite/galera/r/mdev-30013.result

```diff
@@ -0,0 +1,17 @@
+connection node_2;
+connection node_1;
+INSTALL PLUGIN ARCHIVE SONAME 'ha_archive.so';
+CREATE TABLE t (a CHAR(1)) ENGINE=ARCHIVE;
+SHOW CREATE TABLE t;
+Table	Create Table
+t	CREATE TABLE `t` (
+  `a` char(1) DEFAULT NULL
+) ENGINE=ARCHIVE DEFAULT CHARSET=latin1 COLLATE=latin1_swedish_ci
+INSERT INTO t VALUES ('A');
+UNINSTALL SONAME 'ha_archive';
+Warnings:
+Warning	1620	Plugin is busy and will be uninstalled on shutdown
+ALTER TABLE t CHANGE COLUMN a a CHAR(2);
+INSERT INTO t (a) VALUES ('AB');
+ERROR 42000: Unknown storage engine 'ARCHIVE'
```

Comment:
Does the result captured by this test actually demonstrate the expected and desired behavior?

It appears to me that it reveals a further bug, or at least a misleading error message.

The error message says, "Plugin is busy and will be uninstalled _on shutdown_", but then the table is _clearly unusable_ immediately, rather than continuing to work until the engine is shut down.

Reply:
I tested this without Galera and as you can see result is exactly the same:

`SELECT @@global.wsrep_on;
@@global.wsrep_on
0
INSTALL PLUGIN ARCHIVE SONAME 'ha_archive.so';
CREATE TABLE t (a CHAR(1)) ENGINE=ARCHIVE;
SHOW CREATE TABLE t;
Table	Create Table
t	CREATE TABLE `t` (
  `a` char(1) DEFAULT NULL
) ENGINE=ARCHIVE DEFAULT CHARSET=latin1 COLLATE=latin1_swedish_ci
INSERT INTO t VALUES ('A');
UNINSTALL SONAME 'ha_archive';
Warnings:
Warning	1620	Plugin is busy and will be uninstalled on shutdown
ALTER TABLE t CHANGE COLUMN a a CHAR(2);
INSERT INTO t (a) VALUES ('AB');
ERROR 42000: Unknown storage engine 'ARCHIVE'
`

Therefore, while I agree this looks not desired behavior it belong to different MDEV and to MariaDB developers.